### ISSUE TITLE
The toggle button for dark button needs a label.

### ISSUE THREAD

**Issue by PragatiVerma18** [CircuitVerse/CircuitVerseDocs#166]
**Is your feature request related to a problem? Please describe.**

The toggle button on the docs page( https://docs.circuitverse.org/#/ ) did not specify its purpose. Separate it from the edit on GitHub link. It is confusing at first sight.

**Additional context**

![toggle](https://user-images.githubusercontent.com/42115530/67973325-54a45480-fc36-11e9-83c1-7338f9907e2d.PNG)

**Comment by aayush-05** [comment]
I guess you are looking for this @CodeSarthak : https://github.com/CircuitVerse/CircuitVerseDocs/blob/e62b6e8598ccee775a1caeb2ac2d128c62d6e13f/docs/index.html

**Comment by YashKumarVerma** [comment]
Mentioning duplicate Issues for reference
- #213
- #212
- #154

**Comment by nileshdas** [comment]
@YashKumarVerma Should we implement something like this? 
![darkMode](https://user-images.githubusercontent.com/39062229/84751582-5b6fb400-afda-11ea-801c-4f9a2add2d1a.gif)

**Comment by Shivansh2407** [comment]
> @YashKumarVerma Should we implement something like this?
> ![darkMode](https://user-images.githubusercontent.com/39062229/84751582-5b6fb400-afda-11ea-801c-4f9a2add2d1a.gif)

I think we should match it with the CircuitVerse Theme. @YashKumarVerma views.

**Comment by YashKumarVerma** [comment]
> > @YashKumarVerma Should we implement something like this?
> > ![darkMode](https://user-images.githubusercontent.com/39062229/84751582-5b6fb400-afda-11ea-801c-4f9a2add2d1a.gif)
> 
> I think we should match it with the CircuitVerse Theme. @YashKumarVerma views.

Agreed. We have everything in a simple design. @Shivansh2407  how about https://facebook.github.io/watchman/

**Comment by Shivansh2407** [comment]
This looks good but we can implement it after making changes like replacing blue shadow with green to match CV theme. Let me know your views.

**Comment by Yana-do-code** [comment]
Closing as duplicate .
